espnet2/tasks/spk.py

```diff
@@ -252,7 +252,8 @@ def required_data_names(
     def optional_data_names(
         cls, train: bool = True, inference: bool = False
     ) -> Tuple[str, ...]:
-        retval = ("spk_labels",)
+        retval = ("speech2", "trial", "spk_labels")
```

Comment:
what is "speech2"?

Reply:
When calculating EER, we need trials where each trial has two utterances. 
`speech2` is for the second utterance of a trial pair.

Any recommendations for an alternative name?

Reply:
With this explanation in the comment, you can keep using this.

Reply:
ok, then I will add comment on this